src/workers/postUpdated.ts

```diff
@@ -203,6 +204,12 @@ const createPost = async ({
   data.origin = data?.scoutId
     ? PostOrigin.CommunityPicks
     : data.origin ?? PostOrigin.Crawler;
+  data.visible = getPostVisible({ post: data });
+  data.visibleAt = data.visible ? postCreatedAt : null;
+  data.flags = {
+    ...data.flags,
+    visible: data.visible,
+  };
```

Comment:
Because of removal of https://github.com/dailydotdev/daily-api/pull/1608/files#r1427897526 we set the default visible status here during creation.